docs/configuration.rst

```diff
@@ -0,0 +1,141 @@
+.. _configuration:
+
+Configuration Specifications
+============================
+This page documents the various options and
+parameters that can be set in the configuration
+file.
+
+MetSim Section
+--------------
+
+**Required Variables**
+
+``time_step :: int``: The timestep to disaggregate in minutes.  If given as 1440
+(number of minutes in a day), no disaggregation will occur. This value should be
+divide 1440 evenly.
+
+``start :: str``: The time to start simulation given in the format 
+``yyyy/mm/dd`` or ``yyyy/mm/dd:hh``.
+
+``stop :: str``: The time to end simulation given in the format
+``yyyy/mm/dd``.
+
+``forcing :: path``: The path to the input forcing file(s).  See the section 
+on __in_vars__ for more details.
+
+``domain :: path``: The path to the input domain file.  See the section on 
+__domain_vars__ for more details.
+
+``out_dir :: path``: The location to write output to.  If this path doesn't 
+exist it will be created.
+
+``in_format :: str``: A string representing the type of input files specified in
+the ``forcing`` entry.  Can be one of the following: ``ascii``, ``binary``, 
+``netcdf``.
+
+``out_format :: str``: A string representing the type of output to write to 
+``out_dir``.  Can be either ``netcdf`` or ``ascii``.
+
+``method :: str``: A string representing the simulation methods to use.  The
+current implementation only supports ``mtclim``.
+
+``t_max_lr :: float``: The lapse rate (K/m) to be applied to the maximum daily 
+temperature.
+
+``t_min_lr :: float``: The lapse rate (K/m) to be applied to the minimum daily 
+temperature.
+
+**Optional Variables**
+
+``out_prefix :: str``: The output file base name. Defaults to ``forcing``.
```

Comment:
Maybe add the naming convention for output files here? Or maybe you could add it to line 30 above.